workflow-service/src/main/java/com/redhat/parodos/workflow/execution/service/WorkFlowServiceImpl.java

```diff
@@ -68,6 +68,7 @@
 import io.micrometer.core.instrument.Counter;
 import io.micrometer.core.instrument.MeterRegistry;
 import lombok.extern.slf4j.Slf4j;
+import org.jetbrains.annotations.NotNull;
```

Comment:
same. haven't scanned the PR to see if it was injected in other places

Reply:
It was not but I do see this import and the Nullable one in others places

Reply:
I delete them before, but they just keep coming :-) 
We should probably extend the checkstyle with a custom rule to prevent its use. No biggy.